### DISCUSSION TITLE
Next Js with Github Pages

### DISCUSSION THREAD

**Discussion by hggaguilera** [vercel/next.js#15005] [Help]
How can I deploy my next js project on github pages, is there any config I should follow or only need to run yarn next build and it will do it automatically for me?

**Comment by leerob** [comment]
```
next build && next export
```

https://nextjs.org/docs/advanced-features/static-html-export

**Note:** I think GitHub pages uses the `gh-pages` branch [by default](https://docs.github.com/en/github/working-with-github-pages/creating-a-github-pages-site).
